app.rb

```diff
@@ -130,7 +131,11 @@ class App < Sinatra::Base
 			return erb :signup
 		else
 			begin
-				deskpro.post @form.to_ticket
+				if ENV['USE_ZENDESK'] != "true"
```

Comment:
This is confusing logic, you should negate the if condition and replace the if and else blocks.

```ruby
if ENV['USE_ZENDESK'] == "true"
    zendesk.tickets.create! @form.to_zendesk_ticket
else
    deskpro.post @form.to_deskpro_ticket
end
```